airflow/ui/src/layouts/BaseLayout.tsx

```diff
@@ -35,6 +38,19 @@ export const BaseLayout = ({ children }: PropsWithChildren) => {
     document.title = instanceName;
   }
 
+  const [searchParams, setSearchParams] = useSearchParams();
+  const paramToken = searchParams.get("token");
```

Comment:
Should we also use `TOKEN_STORAGE_KEY` here too if they're going to be the same?

Reply:
Done.